<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlStripClass</class>
 <widget class="QWidget" name="ControlStripClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlStrip</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="ListW_OrderList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="LineEditModify" name="LE_Order">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_AddOrder">
       <property name="styleSheet">
        <string notr="true">width: 74px;
height: 21px;
background: #F3F7FD;
border: 1px solid #92A8C6;
border-radius: 2px;

font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #1A1A1A;
line-height: 60px;</string>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEditModify</class>
   <extends>QLineEdit</extends>
   <header>lineeditmodify.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
